dimod/core/polysampler.py

```diff
@@ -51,55 +54,54 @@ def sample_poly(self, polynomial, **kwargs):
         """Sample from a higher-order polynomial."""
         pass
 
-    def sample_hising(self, h, J, **kwargs):
+    def sample_hising(self, h: dict, J: dict, **kwargs) -> SampleSet:
```

Comment:
Should this be `Dict[Tuple[Variable, Variable], Bias]` ?

Reply:
That would be more specific, so yes.